website/docs/docs/collaborate/govern/model-contracts.md

```diff
@@ -98,84 +98,84 @@ When building a model with a defined contract, dbt will do two things differentl
+| not_null        | ✅ | ✅  |
+| primary_key     | ✅ | ❌  |
+| foreign_key     | ✅ | ❌  |
+| unique          | ❌ | ❌  |
+| check           | ❌ | ❌  |
 
 </TabItem>
 <TabItem value="Postgres" label="Postgres">
 
-| Constraint type | Support      | Platform enforcement |
-|:----------------|:-------------|:--------------------|
-| not_null        | ✅  Supported |	✅  Enforced |
-| primary_key     | ✅  Supported |	✅  Enforced |
-| foreign_key     | ✅  Supported |	✅  Enforced |
-| unique          | ✅  Supported |	✅  Enforced |
-| check           | ✅  Supported |	✅  Enforced |
+| Constraint type | Definable     | Enforced |
+|:----------------|:-------------:|:--------------------:|
+| not_null        | ✅  |	✅  |
+| primary_key     | ✅  |	✅  |
+| foreign_key     | ✅  |	✅  |
+| unique          | ✅  |	✅  |
+| check           | ✅  |	✅  |
 
 </TabItem>
 <TabItem value="Spark" label="Spark">
 
-Currently, `not_null` and `check` constraints are supported and enforced only after a model builds. Because of this platform limitation, dbt considers these constraints `supported` but `not enforced`, which means they're not part of the "model contract" since these constraints can't be enforced at build time. This table will change as the features evolve.
+Currently, `not_null` and `check` constraints are enforced only after a model builds. Because of this platform limitation, dbt considers these constraints definable but not enforced, which means they're not part of the "model contract" since these constraints can't be enforced at build time. This table will change as the features evolve.
 
-| Constraint type | Support     | Platform enforcement |
-|:----------------|:------------|:---------------------|
-| not_null        |	✅  Supported | ❌ Not enforced |
-| primary_key     |	✅  Supported | ❌ Not enforced |
-| foreign_key     |	✅  Supported | ❌ Not enforced |
-| unique          |	✅  Supported | ❌ Not enforced |
-| check           |	✅  Supported | ❌ Not enforced |
+| Constraint type | Definable    | Enforced |
+|:----------------|:------------:|:---------------------:|
+| not_null        |	✅  | ❌ |
+| primary_key     |	✅  | ❌ |
+| foreign_key     |	✅  | ❌ |
+| unique          |	✅  | ❌ |
+| check           |	✅  | ❌ |
 
 </TabItem>
 <TabItem value="Databricks" label="Databricks">
 
-Currently, `not_null` and `check` constraints are supported and enforced only after a model builds. Because of this platform limitation, dbt considers these constraints `supported` but `not enforced`, which means they're not part of the "model contract" since these constraints can't be enforced at build time. This table will change as the features evolve.
+Currently, `not_null` and `check` constraints are enforced only after a model builds. Because of this platform limitation, dbt considers these constraints supported but not enforced, which means they're not part of the "model contract" since these constraints can't be enforced at build time. This table will change as the features evolve.
```

Comment:
```suggestion
Currently, `not_null` and `check` constraints are enforced only after a model is built. Because of this platform limitation, dbt considers these constraints definable but not enforced, which means they're not part of the "model contract" since they can't be enforced at build time. This table will change as the features evolve.
```